src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1,3 +1,84 @@
+//! This module contains all filesystem-related system calls.
+//! 
+//! ## File System Calls
+//! 
+//! Cages have methods for filesystem-related calls. They return a code or an error from the `errno` enum.
+//! 
+//! - [open_syscall](crate::safeposix::cage::Cage::open_syscall)
```

Comment:
Note, that Rustdoc will automatically create a list like this for you for each of the functions.  I'm not sure you need to do this manually.

Reply:
So, the sys calls are implemented as part of Cage struct. and we implemented them in different modules for better maintainability. but rustdoc associates these functions to Cage struct and these modules turn up empty. so added these lists to redirect to the Cage docs.

Reply:
Okay, please make sure you indicate this in a comment in the code

Reply:
(By the way, you can look at fdtables, which has a similar issue but gets around it.  This may change the visibility / import structure of the code though.  I honestly, don't recall exactly how I did this.)